src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message should be more informative and indicate that it is an `Excavator` that is working. For example: 'Excavator is working.'